Update KnownWorkingSupportingSoftware.md

snehanaik22 opened the pull request:
Made changes in Latest Known Working Version for Oracle and MS SQL databases

snehanaik22 committed:
Update KnownWorkingSupportingSoftware.md

edalex-ian reviewed (changes requested):
Thanks @snehanaik22 , just looks like you got the numbers in the wrong column. How about you place those ones where the `TBD`s are (Minimum Version), and then the highest versions you tested against into the column (Lastest Known Working Version) you did change?

Also, any chance you could modify the Postgres to be versions 9.6.x to 14.1.x too please?

Thanks.

snehanaik22 committed:
Update KnownWorkingSupportingSoftware.md

As suggested, made changes.

edalex-ian committed:
docs: remove unnecessary +

Doesn't make sense to have a + on the end of the version for the start of the range - especially when an end is specified.

edalex-ian reviewed (approved):
Thanks @snehanaik22 . I just made a small tweak, and will now merge. :+1:
